### PR TITLE
Replace American spelling with British spelling

### PR DESCRIPTION
**Pull request by rprimus**
- [X] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md) document?
- [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [X] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/master/Library/Homebrew/test/PATH_spec.rb).
- [X] Have you successfully run `brew style` with your changes locally?
- [X] Have you successfully run `brew tests` with your changes locally?

-----
Wed Oct 30 18:05:17 GMT 2019

tl;dr:  -ize -> -ise

This PR replaces words spelt in American english with their British equivalents.

### CODE REVIEW

**Comment by rprimus** [comment]
Wed Oct 30 18:13:22 GMT 2019

@dawidd6 

https://github.com/Homebrew/brew/blob/eda0220e5f807482e4feb327e34375de308ea0fc/docs/Prose-Style-Guidelines.md#style-and-usage

**Comment by reitermarkus** [comment]
You'll need to run `brew man`.
